plugins/performance-lab/includes/admin/plugins.php

```diff
@@ -19,13 +19,18 @@
  * @return array{name: string, slug: string, short_description: string, requires: string|false, requires_php: string|false, requires_plugins: string[], download_link: string, version: string}|WP_Error Array of plugin data or WP_Error if failed.
  */
 function perflab_query_plugin_info( string $plugin_slug ) {
-	$transient_key = 'perflab_plugins_info-v2';
+	$transient_key = 'perflab_plugins_info';
 	$plugins       = get_transient( $transient_key );
 
 	if ( is_array( $plugins ) ) {
 		// If the specific plugin_slug is not in the cache, return an error.
 		if ( ! isset( $plugins[ $plugin_slug ] ) ) {
-			return new WP_Error( 'plugin_not_found', __( 'Plugin not found.', 'performance-lab' ) );
+			return new WP_Error(
+				'plugin_not_found',
+				__( 'Plugin not found.', 'default' ) . ' ' .
+				/* translators: %s is error code */
+				sprintf( '(Error code: %d)', __LINE__ )
```

Comment:
I find the "error code" terminology confusing. It makes me think this is either an HTTP response code or an actual error code I can look up in some docs. Why not say "line"?

Reply:
Sure. Done in 49cb889ca